src/TornCityBot.user.js

```diff
@@ -4,136 +4,231 @@
 // @version      0.6.1
 // @description  Try to take over the world, for the lazy.
 // @author       Jurriaan Den Toonder<jurriaan.toonder@liquidpineapple.net>
-// @downloadURL  https://github.com/Fastjur/Torn-City-Bot/raw/master/src/TornCityBot.user.js
-// @updateURL    https://github.com/Fastjur/Torn-City-Bot/raw/master/src/TornCityBot.user.js
-// @match        https://*.torn.com/*
```

Comment:
The script is currently not allowed to run on the required pages, why have you taken this out?